src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -8312,7 +8367,7 @@ namespace OutputReportTabular {
             leedSiteSrvWatr = 0.0;
             leedSiteRecept = 0.0;
             leedSiteTotal = 0.0;
-            for (iResource = 1; iResource <= 5; ++iResource) { // don't bother with water
+            for (iResource = 1; iResource <= 12; ++iResource) { // don't bother with water
```

Comment:
12 again here.

Reply:
It is 12 here because this FOR loop doesn't count water.